record_indexer/delete_index.py

```diff
@@ -0,0 +1,26 @@
+import sys, os
+import settings
+import requests
+import argparse
+
+ENDPOINT = os.environ.get('RIKOLTI_ES_ENDPOINT')
+AUTH = ('rikolti', os.environ.get('RIKOLTI_ES_PASS'))
```

Comment:
I've moved these into settings. I've also updated all use of ENDPOINT to use fstrings, so `url=f"{settings.ENDPOINT}/{index}"` - they shouldn't use `os.path.join` because they're not operating system paths, they're urls, which universally use a `/` character and don't need the operating system interface provided by the builtin `os` module!

Reply:
Thanks for making these changes. Totally makes sense to import from settings. And also yeah, no reason at all to use `os.path.join` here!